jenkins/sparklingWaterPipeline.groovy

```diff
@@ -257,20 +242,24 @@ def prepareSparkEnvironment() {
 def prepareSparklingWaterEnvironment() {
     return { config ->
         stage('QA: Prepare Sparkling Water Environment - ' + config.backendMode) {
-            sh """
-                if [ ${config.backendMode} = external ]; then
-                    if [ ${config.buildAgainstH2OBranch} = true ]; then
-                        # In this case, PySparkling build is driven by H2O_HOME property
-                        # When extending from specific jar the jar has already the desired name
-                        ${getGradleCommand(config)} -q :sparkling-water-examples:build -x check -PdoExtend extendJar
-                        # Copy also original driver JAR to desired location so rest API based client can use it
-                        cp ${env.H2O_ORIGINAL_JAR} ${env.H2O_DRIVER_JAR}
-                    else
-                        cp `${getGradleCommand(config)} -q :sparkling-water-examples:build -x check -PdoExtend extendJar -PdownloadH2O=${config.driverHadoopVersion}` ${env.H2O_EXTENDED_JAR}
-                        cp `${getGradleCommand(config)} -q -PdoExtend -x check -PdownloadH2O=${config.driverHadoopVersion} getDownlodedJarPath` ${env.H2O_DRIVER_JAR}
-                     fi
-                fi
-                """
+            if (config.backendMode.toString() == "external") {
+                if (config.buildAgainstH2OBranch.toBoolean()) {
+                    sh """
+                        git clone https://github.com/h2oai/h2o-3.git
+                        cd h2o-3
+                        git checkout ${config.h2oBranch}
+                        . /envs/h2o_env_python2.7/bin/activate
```

Comment:
Shouldn't the same be also done for `. /envs/h2o_env_python3.6/bin/activate`?

Reply:
Nope, we just need any python, but we don't later depend on any specific python version as we just copy the python sources directly